db/migrate/20190214125845_add_comments_to_miq_tasks_table.rb

```diff
@@ -0,0 +1,21 @@
+class AddCommentsToMiqTasksTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :miq_tasks, "ManageIQ background tasks"
+
+    change_column_comment :miq_tasks, :context_data, "Serialized metadata that may contain more detailed information about the task."
+    change_column_comment :miq_tasks, :created_on, "The timestamp the task was added to the app inventory."
+    change_column_comment :miq_tasks, :id, "The internal database ID."
+    change_column_comment :miq_tasks, :identifier, "The resource this task is related to in '#{klass}:#{id}' format."
+    change_column_comment :miq_tasks, :message, "Message that describes the reason for the current status."
```

Comment:
this message shown in UI on task management screen

Reply:
also, message could be updated independently from status

Reply:
Many fields are shown within the UI, so I'm not sure I want to add that note explicitly here. As for the message contents, how you would you more generically describe it then?

Reply:
I think this column `message` should have only one responsibility (compare to other many field) - to be shown in log or in UI  (however some may use it for different purposes)

Reply:
Alright how does, "Message associated with the task for logging or UI purposes" sound?

Reply:
👍

Reply:
Ok, updated.